### PR TITLE
Job Page w/o api call

### PR DESCRIPTION
**Pull request by mpolimen**
Test it on your own going to /test_job

### CODE REVIEW

**Comment by ryan280** [comment]
The menu tabs on smaller devices will often cut off the tab on the edge, especially the details tab. 

Also would suggest that the Role, and Company name change dynamically with device size if there is an easy way to do this. Not a big deal though.

File: client/src/pages/Job/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+export default function Job() {
+    // Constants
+    const BASE_CLEARBIT_URL = 'https://logo.clearbit.com/';
+
+    // State variables
+    const [currTab, setCurrTab] = useState(0);
+    const [role, setRole] = useState("Software Engineer");
+    const [company, setCompany] = useState("Google");
+    const [url, setUrl] = useState("google.com");
```
Comment (bwormser) [inline client/src/pages/Job/index.tsx]: Is there a reason to set these initially?

**Review by bwormser — COMMENTED** [review]
If the set values are just testing until the API call works then LGTM

File: client/src/pages/Job/index.tsx (hunk 1 above)
Comment (mpolimen) [reply client/src/pages/Job/index.tsx]: Just to give them clear types like string or number

Comment (mpolimen) [reply client/src/pages/Job/index.tsx]: But no, they can be generic

**Comment by GraceMattern** [comment]
Can you add some margins (left and top) on the Header because we need room for the drawer

**Pull request merged**